lib/reevoo_mark_utils.php

```diff
@@ -30,7 +30,10 @@ function getTrkref($options = array()) {
   }
 
   function getPaginationParams($options = array()) {
-    $page = self::presenceKey($_GET, 'reevoo_page', 1);
+    $page = isset($options['page'])
```

Comment:
I think this is good, it just adding the ability to override the default "reevoo_" parameters, if the retailer doesn't want to use the override it will default to the current "reevoo_" ones so all it's good. It just gives more flexibility to the client which is good. But I'm width Ed we will need to add specs to this before merging.